apps/i18n/spritelab/en_us.json

```diff
@@ -49,6 +49,7 @@
   "outbreakSuccess": "You've finished the level. When you are ready for the next challenge, press Continue.",
   "noSprites": "You need to make a sprite.",
   "noBehavior": "Your sprite needs a behavior.",
+  "noNewBehavior": "Your sprite need didn't begin a new behavior.",
```

Comment:
i might just not understand the context, but this string doesn't make sense to me

Reply:
oh good catch, thanks! I just copied them from what Mike gave me, but I should have looked closer.
Fixed.